### PR TITLE
Added extra parameters to TextSummaryTask

### PR DESCRIPTION
**Pull request by ashkaaar**
Added the parameters regarding the issue https://github.com/griptape-ai/griptape/issues/203

length
target audience
format

Unit tests: passed
waiting for review and suggestions

### CODE REVIEW

File: griptape/engines/summary/base_summary_engine.py [hunk 1]
```diff
@@ -2,12 +2,11 @@
 from attr import define
 from griptape.artifacts import TextArtifact, BaseArtifact
 
-
 @define
 class BaseSummaryEngine(ABC):
-    def summarize_text(self, text: str) -> str:
-        return self.summarize_artifacts([TextArtifact(text)]).value
+    def summarize_text(self, text: str, length: int = None, target_audience: str = None, format: str = None) -> str:
```
Comment (vasinov) [inline griptape/engines/summary/base_summary_engine.py:7]: Those should be class properties for:
- DRYness across class methods and
- ease of integration with tasks and `TextMemoryActivitiesMixin.summarize`.

Comment (vasinov) [reply griptape/engines/summary/base_summary_engine.py:7]: Additionally, we always use `Optional`s in type hints if the default value is `None`. In this case it should be `Optional[int]`, `Optional[str]`, and `Optional[str]`.

**Review by vasinov — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by collindutter** [comment]
@ashkaaar I am going to close this PR in favor of the work that will be done on https://github.com/griptape-ai/griptape/issues/342. Implementing this functionality as described in the other issue gives users more control over the output, and it provides a more consistent experience across Tasks.

**Comment by collindutter** [comment]
No problem at all! Thanks for the original PR 🙂

**Pull request closed without merging**